### ISSUE TITLE
Support saving all meshes (currently you have to select them individually and save manually)

### ISSUE THREAD

**Issue by silcowitz** [CloudCompare/CloudCompare#569]
This is a hassle if you have many of them

**Comment by dgirardeau** [comment]
Do you mean that you expect CC to save the N meshes in N different files automatically? How would you specify the different filenames?

**Comment by silcowitz** [comment]
yeah I see your point. I just bumped into a case where I had a bunch of meshes imported, that I had done a number of ops on, and I wanted to save them all back again, to the same name/format as they had when I imported them.
